src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterTests.cs

```diff
@@ -496,6 +496,45 @@ void Method()
                 expectedIndentation: 20);
         }
 
+        [WpfFact]
+        [Trait(Traits.Feature, Traits.Features.SmartIndent)]
+        public void ExtendedPropertyPattern_Before()
+        {
+            var code = @"
+class C
+{
+    void M()
+    {
+        _ = this is
+        {
+
+";
+            AssertSmartIndent(
+                code,
+                indentationLine: 7,
+                expectedIndentation: 12);
+        }
+
+        [WpfFact]
+        [Trait(Traits.Feature, Traits.Features.SmartIndent)]
+        public void ExtendedPropertyPattern_After()
+        {
+            var code = @"
+class C
+{
+    void M()
+    {
+        _ = this is
+        {
+            A.B: 1,
+
+";
+            AssertSmartIndent(
+                code,
+                indentationLine: 8,
+                expectedIndentation: 12);
+        }
```

Comment:
can you add a test with this form:

```c#
class C
{
    void M()
    {
        _ = this is
        {

            A.B: 1,
```
Where we're testing the indentation *above* `A.B: 1`

Reply:
Adjusted test to cover this (I think)